### DISCUSSION TITLE
[Feature Request] Add `Line_Cell_Circulator` similar to `Line_Face_Circulator` in 3D triangulation

### DISCUSSION THREAD

**Discussion by ImperatorS79** [CGAL/cgal#8443] [Q&A] [answered]
In 2D triangulation, starting from a face f, it is possible to obtain the sequence of faces intersected by a line p-q using the `Line_Face_Circulator`.
In 3D triangulation, starting from a cell c, no such class exists to obtain the sequence of cells intersected by a line p-q.

**Comment by janetournois** [comment, marked as answer]
In a 3D triangulation, you can use the `Segment_cell_iterator` and `Segment_simplex_iterator` to collect the sequence of intersected cells (resp. simplices)

**Reply by ImperatorS79** [reply]
Great, it was exactly what I was looking for.
